### PR TITLE
Feat/override plugins on install

### PR DESCRIPTION
**Pull request by ElenaDiachenko**
## Description

- override_plugins on install in new project

## Related issues

- GH issues

## Npm releases

n/a

## Testing 

- `yarn bootstrap-clean`
-

### CODE REVIEW

File: packages/template-starter/scripts/postinstall.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                        }
+                    });
+                }
+            }
+        });
+        removeDirSync(overrideDir);
+        return true;
+    }
+    return false;
+};
```
Comment (RicardasN) [inline packages/template-starter/scripts/postinstall.js:75]: Not entirely sure I understand why we're reverting overrides after post install, what's the idea behind this logic?

Comment (ElenaDiachenko) [reply packages/template-starter/scripts/postinstall.js:75]: @RicardasN For example, I reinstall or update a plugin that needs to be overridden, but I still have the old override in the .rnv/overrides folder.

Comment (ElenaDiachenko) [reply packages/template-starter/scripts/postinstall.js:75]: And one more thing. I'm not sure if it's right that the script is run multiple times during rnv new

Comment (RicardasN) [reply packages/template-starter/scripts/postinstall.js:75]: I think it would be better to avoid it being run multiple times, but it is fine

**Review by RicardasN — APPROVED** [review]

**Comment by GabrieleKaceviciute** [comment]
@ElenaDiachenko as you said, I've tried to create new project but i'm getting this error -> 
[new.txt](https://github.com/user-attachments/files/17241124/new.txt)
```
➜  renative git:(feat/override_plugins_on_install) git status
On branch feat/override_plugins_on_install
Your branch is up to date with 'origin/feat/override_plugins_on_install'.

nothing to commit, working tree clean
➜  renative git:(feat/override_plugins_on_install)
```

**Review by GabrieleKaceviciute — APPROVED** [review]

**Pull request merged**